packages/core/metrics/core-metrics-server/src/metrics.ts

```diff
@@ -92,7 +113,7 @@ export interface OpsProcessMetrics {
   /** node event loop delay histogram since last collection */
   event_loop_delay_histogram: IntervalHistogram;
   /** node event loop utilization since last collection */
-  event_loop_utilization: EventLoopUtilization;
+  event_loop_utilization: EventLoopUtilizationWithLoad;
```

Comment:
This adds the "load" concept to the existing `event_loop_utilization` which avoids creating another API but expands our public API.